guides/source/active_record_querying.md

```diff
@@ -673,7 +673,7 @@ GROUP BY date(created_at)
 HAVING sum(price) > 100
 ```
 
-This will return single order objects for each day, but only those that are ordered more than $100 in a day.
+This will return single order objects for each day, but only days in which more than $100 was ordered.
```

Comment:
I'm having a hard time understand this sentence (and the original version was even harder).

Is this returning an array of orders?